fe/src/app/core/home/home.component.scss

```diff
@@ -0,0 +1,10 @@
+.container {
```

Comment:
Need to remove some of this duplication. Perhaps it would be better to put styles like this (i.e. that are repeated) inside of: `styles.scss`?